pageserver/src/pgdatadir_mapping.rs

```diff
@@ -499,6 +499,24 @@ impl Timeline {
         self.get(CHECKPOINT_KEY, lsn, ctx).await
     }
 
+    pub async fn list_aux_files(
+        &self,
+        lsn: Lsn,
+        db_id: Oid,
```

Comment:
Repl slots are global per installation, not per db. So this would break if you try to create slot in one database and drop in another; or if slot is created, but later database is deleted. So let's make aux files global here as well?

Reply:
Yes, makes sense.
I didn't realised that logical slot are not db specific.
Done